migrations/0014_add_access_groups_to_queries.py

```diff
@@ -0,0 +1,17 @@
+import peewee
+from playhouse.migrate import PostgresqlMigrator, migrate
+from redash import models
+from redash.models import db
+
+
+if __name__ == '__main__':
+    db.connect_db()
+    migrator = PostgresqlMigrator(db.database)
+
+    with db.database.transaction():
+        migrate(
+            migrator.add_column('queries', 'access_groups', models.Query.access_groups),
+        )
+        models.Query.update(access_groups=['default']).execute()
```

Comment:
So the thing is, if you specified `default` value, and then run `\d+ queries` in Postgres, you will see it didn't actually apply default value for `ArrayField`. I suspect this is a bug of Playhouse extensions.